### ISSUE TITLE
`name` for `valueCount` doesn't work

### ISSUE THREAD

**Issue by ad-si** [pola-rs/nodejs-polars#268]
#### Have you tried latest version of polars?

- [yes]

#### What version of polars are you using?

0.15

#### What operating system are you using polars on?

macOS Sonoma 14.6.1 arm64

#### What node version are you using

bun 1.1.27

#### Describe your bug

`name` for `valueCount` does not change the `count` column name.

#### What are the steps to reproduce the behavior?

```js
import pl, { DataType } from "nodejs-polars"

const chars = pl.Series("char", ["a", "b", "b", "c"])

console.log(chars.valueCounts(true, true, "xxx"))
```

#### What is the actual behavior?

```txt
shape: (3, 2)
┌──────┬────────────┐
│ char ┆ proportion │
│ ---  ┆ ---        │
│ str  ┆ u32        │
╞══════╪════════════╡
│ b    ┆ 2          │
│ a    ┆ 1          │
│ c    ┆ 1          │
└──────┴────────────┘
```

#### What is the expected behavior?

```txt
shape: (3, 2)
┌──────┬─────┐
│ char ┆ xxx │
│ ---  ┆ --- │
│ str  ┆ u32 │
╞══════╪═════╡
│ b    ┆ 2   │
│ a    ┆ 1   │
│ c    ┆ 1   │
└──────┴─────┘
```

**Comment by Bidek56** [comment]
This is fixed in #267 Thanks for finding this issue!

**Comment by Bidek56** [comment]
Fix was merged into main. Thx
